e2e/editor-drawer.spec.ts

```diff
@@ -100,9 +101,9 @@ test.describe("feature image", () => {
     // Check that the post was saved successfully
     const saveNotificationTitle = page.locator("#toast-1-title");
     await expect(saveNotificationTitle).toBeVisible();
-    expect(await saveNotificationTitle.innerText()).toBe(
-      "Post has been updated.",
-    );
+    await expect(saveNotificationTitle).toHaveText("Post has been updated.", {
+      useInnerText: true,
+    });
```

Comment:
```suggestion
    await expect(saveNotificationTitle).toHaveText("Post has been updated.");
```
This seemed to work just fine for me. Was it necessary to have `useInnerText` on your machine?

Reply:
Not exactly. Playwright dint work properly when I checked yesterday so I pushed them here to check them through actions. Lemme clean up and try the tests again.